specification/maps/data-plane/Creator/readme.go.md

```diff
@@ -0,0 +1,26 @@
+## Go
+
+These settings apply only when `--go` is specified on the command line.
+
+``` yaml $(go)
+go:
+  license-header: MICROSOFT_MIT_NO_VERSION
+  namespace: azmaps.creator
```

Comment:
@propheel: go doesn't really go well with dotted packages, you will go generating azmaps.creator folder with source files with package azmapscreator, that won't import unless folder is also renamed to `azmapscreator`

Also `azmaps` can be redundant, the import would then look like:

```
import (
	"github.com/Azure/azure-sdk-for-go/services/preview/maps/preview/2.0/azmapscreator"
```

which probably will look better like: `github.com/Azure/azure-sdk-for-go/services/preview/maps/preview/2.0/creator`

For reference, cognitive services(which are also split) imports look like:

```
"github.com/Azure/azure-sdk-for-go/services/cognitiveservices/v1.0/customsearch"
```